ci/debian.cross-compile.sh

```diff
@@ -22,6 +22,9 @@ apt install -y --no-install-recommends \
 	libftdi-dev:${ARCH} \
 	libudev-dev:${ARCH} \
 	libyaml-dev:${ARCH} \
+	pkg-config:${ARCH} \
```

Comment:
Quite no. You are installing an executable package for the target architecture. Quite a hack I'd say.

Reply:
Let me check if there’s not a similar package as gcc

Reply:
Currently `pkg-config` is `Multi-Arch:same`, which means that is is arch-independent. So you can do it, but I'd still consider for it to be a hack ;-)